core/src/main/resources/lib/layout/html.jelly

```diff
@@ -161,6 +161,8 @@ THE SOFTWARE.
       <script src="${resURL}/scripts/msie.js" type="text/javascript"/>
     </j:if>
 
+    <!-- Load vendor scripts common to all JS modules -->
```

Comment:
Maybe make clear that this file is generated during the build.

Reply:
I have updated the comment on https://github.com/jenkinsci/jenkins/pull/4425/commits/99832f1b9460d4c60cd4ca84762adebba0aacdc6 .

Marking it as resolved. Feel free to unresolve it if you'd like further change!